### PR TITLE
minor typos

### PR DESCRIPTION
**Pull request by mvolkmann**
I think this is my first pull request to this repository. Just setting myself up for more in the future.

### CODE REVIEW

File: CoreUpdates/6020-String-RemoveClassVars-JuanVuletich-2023Sep11-15h35m-jmv.001.cs.st [hunk 1]
```diff
@@ -176,7 +176,7 @@ findStringIgnoreCase: subString in: body startingAt: start
 
 !String class methodsFor: 'services' stamp: 'jmv 9/11/2023 14:23:42'!
 is: aStringOrByteArray equalTo: another
-	"Take adventage of String primitive for comparing, for example, ByteArrays"
+	"Take advantage of String primitive for comparing, for example, ByteArrays"
```
Comment (mvolkmann) [inline CoreUpdates/6020-String-RemoveClassVars-JuanVuletich-2023Sep11-15h35m-jmv.001.cs.st:179]: Is it worthwhile to update old change sets?

File: CuisImage/Cuis7.0.sources [hunk 2]
```diff
@@ -2095,8 +2095,8 @@ Refactoring subclass: #AddInstanceVariable
 	^classToRefactor ! !
 
 !AddInstanceVariable methodsFor: 'accessing' stamp: 'HAW 6/6/2017 10:10:40'!
-           newVariable
```
Comment (mvolkmann) [inline CuisImage/Cuis7.0.sources:2098]: I don't know what Git thinks changed here.

File: CuisImage/Cuis7.0.sources [hunk 3]
```diff
@@ -28987,7 +28987,7 @@ self flag: #unicodeNeedsImprovement.
            cr
 	"Answer the Character representing a carriage return."
 
-	^ ($)! !
+	^ ($)! !
```
Comment (mvolkmann) [inline CuisImage/Cuis7.0.sources:28990]: Likewise, I don't know what Git thinks changed here.

File: CuisImage/Cuis7.0.sources [hunk 4]
```diff
@@ -155887,7 +155887,7 @@ String findSubstring: 'abc' in: 'abcdefABcd' startingAt: 2 matchTable: CaseInsen
 
 !String class methodsFor: 'services' stamp: 'jmv 9/11/2023 14:23:42'!
                      is: aStringOrByteArray equalTo: another
-	"Take adventage of String primitive for comparing, for example, ByteArrays"
+	"Take advantage of String primitive for comparing, for example, ByteArrays"
```
Comment (mvolkmann) [inline CuisImage/Cuis7.0.sources:155890]: This is the main change in this PR.

**Comment by jvuletich** [comment]
Older numbered change sets can not be modified after the fact. Cuis7.0.sources should not be edited by hand. This is not the way to publish contributions to the Cuis base image. Please read Documentation/AboutCuis.md.
I pushed an update to correct the typo you found, though.

**Pull request closed without merging**